ramls/samples/profiles/mappingProfile.sample

```diff
@@ -0,0 +1,33 @@
+{
+   "id":"3798f546-7afb-11ea-bc55-0242ac130003",
+   "name":"Example mapping profile",
+   "description":"Example mapping profile description",
+   "recordTypes":[
```

Comment:
If the rules just have Instance Transformation, then the record types must be just INSTANCE, and not the others

Reply:
left only INSTANCE type